.github/workflows/upgrade-current-deployment-infra.yml

```
@@ -0,0 +1,80 @@
+            TF_VAR_BLUE_DATALAYER_HASURA_ADMIN_SECRET: ${{ secrets.DATALAYER_HASURA_ADMIN_SECRET }}
+            #Coingecko API
+            TF_VAR_GREEN_COINGECKO_API_KEY: ${{ secrets.COINGECKO_API_KEY }}
+            TF_VAR_BLUE_COINGECKO_API_KEY: ${{ secrets.COINGECKO_API_KEY }}
+        steps:
+            - name: Check out github repository
+              uses: actions/checkout@v4
+              with:
+                  fetch-depth: 1
+
+            - name: Check if user is an admin
+              uses: ./.github/actions/check-admin
```

actionlint (1.7.4)

description is required in metadata of "Check Admin" action at "/home/jailuser/git/.github/actions/check-admin/action.yml"

(action)